pkg/agent/openflow/client.go

```diff
@@ -468,23 +468,22 @@ func (c *client) InstallClusterServiceFlows() error {
 	return nil
 }
 
-func (c *client) InstallClusterServiceCIDRFlows(serviceNet *net.IPNet, gatewayMAC net.HardwareAddr, gatewayOFPort uint32) error {
-	flow := c.serviceCIDRDNATFlow(serviceNet, gatewayMAC, gatewayOFPort)
-	if err := c.ofEntryOperations.Add(flow); err != nil {
+func (c *client) InstallClusterServiceCIDRFlows(serviceNets []*net.IPNet, gatewayMAC net.HardwareAddr, gatewayOFPort uint32) error {
+	flows := c.serviceCIDRDNATFlow(serviceNets, gatewayMAC, gatewayOFPort)
+	if err := c.ofEntryOperations.AddAll(flows); err != nil {
 		return err
 	}
-	c.defaultServiceFlows = []binding.Flow{flow}
+	c.defaultServiceFlows = append(c.defaultServiceFlows, flows...)
```

Comment:
Better to use c.defaultServiceFlows = flows

Reply:
No, we can't use "c.defaultServiceFlows = flows", some other functions also generate flows which should be included in "defaultServiceFlows". It will override others if we use "=".

Reply:
Done.